.gitpod.yml

```diff
@@ -19,9 +21,8 @@ github:
     addComment: false
     addBadge: true
 
-image:
-  file: .gitpod.dockerfile
+image: gitpod/workspace-mongodb
 
 vscode:
   extensions:
-    - "asciidoctor.asciidoctor-vscode"
+    - bradlc.vscode-tailwindcss
```

Comment:
I also removed the `asciidoctor` extension, because we don't use this anymore.
Other than that I added the tailwind extension, as we bring this in with nextjs, so people have intellisense for the classes.